### PR TITLE
Shutdown by hostname command

### PR DESCRIPTION
**Pull request by benrudolph**
@gcapalbo this is what the command would look like to shutdown a worker by hostname. fine to merge, will do some testing on staging

### CODE REVIEW

**Comment by benrudolph** [comment]
(killed build)

**Comment by gcapalbo** [comment]
looks good. is there any output we can get from this to tell if the command is sent properly or not? i think it would be great if running this command gives a different exit code based on whether the operation was successful or not

**Comment by benrudolph** [comment]
that's a good idea, i'll look into what it returns / if we can make it return something different

File: corehq/apps/hqadmin/management/commands/shutdown_celery_worker_by_hostname.py [hunk 1]
```diff
@@ -11,3 +13,9 @@ def handle(self, hostname, *args, **options):
         celery = Celery()
         celery.config_from_object(settings)
         celery.control.broadcast('shutdown', destination=[hostname])
+        worker_responses = celery.control.ping(timeout=10, destination=[hostname])
+        pings = parse_celery_pings(worker_responses)
+        if hostname in pings:
+            print 'Did not shutdown worker'
+            exit(1)
+        print 'Successfully initiated warm shutdown'
```
Comment (benrudolph) [inline corehq/apps/hqadmin/management/commands/shutdown_celery_worker_by_hostname.py]: i tested this out on staging and `shutdown` returns nothing in any situation. the ping response will return empty though if it's properly in warm shutdown

**Pull request merged**